chapter_09/9-4.py

```diff
@@ -0,0 +1,64 @@
+class Restaurant:
+    """A basic model for a restaurant."""
+
+    def __init__(self, restaurant_name, cuisine_type):
+        """Initialize restaurant's name and cuisine."""
+        self.restaurant_name = restaurant_name
+        self.cuisine_type = cuisine_type
+        self.number_served = 0
+
+    def describe_restaurant(self):
+        """Print a description for the information about the restaurant"""
+        print(
+            f"{self.restaurant_name.title()} serves {self.cuisine_type.title()}."
+        )
+
+    def open_restaurant(self):
+        """Print a statement simulating the restaurant opening."""
+        print(f"{self.restaurant_name.title()} is now open!")
+
+    def patrons_served(self):
+        """Print statement for amount of patrons served."""
+        print(f"We have now served {self.number_served} patrons today.")
+
+    def set_number_served(self, patrons):
+        """Set the number of patrons served to a specific value.
+
+        :param patrons:
```

Comment:
If you're going to use parameter notation, you need to complete it. For example:
```suggestion
        :param patrons: number of patrons
        :type patrons: int
```
This allows hints in IDEs to show users a description of each parameter and the type they expect.

Reply:
Oh, that was from the Pep8 bot, so I thought it was supposed to be like that. Is my original correct? I had put: 

def set_number_served(self, patrons):
        """Set the number of patrons served to a specific value."""
        self.number_served = patrons